includes/class-omise-setting.php

```diff
@@ -7,11 +7,37 @@
 }
 
 class Omise_Setting {
```

Comment:
Should really be called "Settings" - otherwise the name suggests a class that deals with one setting only